docs/examples/grid/index.html

```diff
@@ -106,7 +106,7 @@
       <p></p>
       <div class="row">
         <div class="col-xs-12 col-sm-6 col-lg-8">.col-xs-12 .col-sm-6 .col-lg-8</div>
-        <div class="col-xs-6 col-lg-4">.col-xs-6 .col-lg-4</div>
+        <div class="col-sm-6 col-lg-4">.col-xs-6 .col-lg-4</div>
```

Comment:
the visible text in between the `<div></div>` also needs to be updated, otherwise LGTM good catch